docs/snippets/chains/API-reference.mdx

```diff
@@ -0,0 +1,572 @@
+# API Reference
+
+## *class* `truss_chains.Assets`
```

Comment:
is there any way we could group these better? It would be nice if all of the config-related classes/methods were grouped together here

Reply:
Great improvement idea... I just hope I can get this switched now without too much hassle with the patch file...